### ISSUE TITLE
Publication status messages are visible

### ISSUE THREAD

**Issue by gjost** [densho/encyc-rg#46]
Publication status messages from editors' wiki are visible:
`This page is complete and will be published to the production Encyclopedia.`

**Comment by GeoffFroh** [comment]
* can also hide from ui with css

**Comment by gjost** [comment]
```
<div class="alert alert-success published">
<p>
This page is complete and will be published to the production Encyclopedia.
</p>
</div>
```

**Comment by GeoffFroh** [comment]
fixed with css hack in 2309426f1e7b16e56faa7995d24ec2a0f30233e3
